akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -22,38 +22,10 @@ to see what this looks like in practice.
 
```

Comment:
not changed here but I saw something wrong in above
```
For the new API see @ref:[cluster-sharding](typed/cluster-sharding.md).
```

It should be `[Cluster Sharding]`

Reply:
I think that doesn't matter though, and that the link text is replaced with the text from the ref anyway

Reply:
Fixed thanks.

Reply:
> the link text is replaced with the text from the ref anyway

no, it's not. That is only for ToC links. (and it's very good that it's possible to have another text that fits in with the sentence of the link)